docs/operating-eck/installing-eck.asciidoc

```diff
@@ -140,13 +139,14 @@ kubectl delete -n elastic-system \
     clusterrole.rbac.authorization.k8s.io/elastic-operator-view \
     clusterrole.rbac.authorization.k8s.io/elastic-operator-edit \
     clusterrolebinding.rbac.authorization.k8s.io/elastic-operator \
-    rolebinding.rbac.authorization.k8s.io/elastic-operator \
     service/elastic-webhook-server \
+    configmap/elastic-operator \ <1>
     statefulset.apps/elastic-operator \
     validatingwebhookconfiguration.admissionregistration.k8s.io/elastic-webhook.k8s.elastic.co
 ----
+<1> If you have customized the operator configuration you can keep the operator configuration in this ConfigMap and annotate and label it like the CRDs above.
 
 . Install the ECK operator using the Helm chart as described in <<{p}-install-helm>>.
 
-NOTE: A shell script is available in the link:{eck_github}/blob/{eck_release_branch}/deploy/helm-migrate.sh[ECK source repository] to demonstrate how to migrate from version 1.2.1 to Helm. You can modify it to suit your own environment. 
+NOTE: A shell script is available in the link:{eck_github}/blob/{eck_release_branch}/deploy/helm-migrate.sh[ECK source repository] to demonstrate how to migrate from version 1.7.1 to Helm. You can modify it to suit your own environment.
```

Comment:
I think we should move this note at the top of the section. I bet a lot of people won't read the whole page before they start running the migration commands.